database/engine/rrdengineapi.c

```diff
@@ -191,8 +191,7 @@ void rrdeng_store_metric_flush_current_page(RRDDIM *rd)
         return;
     }
     if (likely(descr->page_length)) {
-        int page_is_empty;
-
+        int /*ret, */page_is_empty;
```

Comment:
@amoss the variable `ret` was removed with the PR https://github.com/netdata/netdata/pull/9845, please, rebase your PR.

Reply:
I already rebased to remove the conflict. This PR will be adding it back in as comment. This is because the code that uses it is still there commented out until we decide on a more permanent fix for the flushing problems.